### PR TITLE
[Elasticsearch-sink] support multi Elasticsearch hosts

### PR DESCRIPTION
**Pull request by wangjialing218**
### Motivation

Current Elasticsearch-sink do not support multi Elasticsearch hosts like "http://172.16.0.1:9200,172.16.0.2:9200"

### Modifications

Add support for multi Elasticsearch hosts

### CODE REVIEW

**Review by sijie — APPROVED** [review]

**Comment by sijie** [comment]
@freeznet @nlu90 @tuteng can you review this?

**Review by tuteng — COMMENTED** [review]
Can you add an integration test? https://github.com/apache/pulsar/blob/master/tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sinks/ElasticSearchSinkTester.java

**Comment by wangjialing218** [comment]
> Can you add an integration test? https://github.com/apache/pulsar/blob/master/tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sinks/ElasticSearchSinkTester.java

@tuteng `ElasticSearchSinkTester` use `ElasticSearchContainer` to preform test. The container is one node es, it seems not possible to perfrom multi nodes test there.
I have add the test in `ElasticSearchSinkTests`, and made `ElasticSearchSink.getClient` from private to protected.

**Review by tuteng — APPROVED** [review]

**Review by freeznet — APPROVED** [review]
LGTM

File: pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchSink.java [hunk 1]
```diff
@@ -139,11 +137,23 @@ private CredentialsProvider getCredentialsProvider() {
         return credentialsProvider;
     }
 
-    private RestHighLevelClient getClient() throws MalformedURLException {
+    private HttpHost[] getHttpHost() {
```
Comment (nlu90) [inline pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchSink.java:140]: nitpick: `getHttpHosts`

**Review by nlu90 — APPROVED** [review]

**Pull request merged**